src/main/perl/TextRender.pm

```diff
@@ -154,16 +153,10 @@ Java properties format (using C<Config::Properties>),
 
 .INI format (using C<Config::Tiny>)
 
-=item general
-
-(using C<Config::General>).
-
-Caution: there is no way to garantee reproducible results as
-there is no control over the order of hash keys. It is recommended to
-try to generate this format in another way (e.g. via TT).
-
 =back
 
+(Builtin module <general> is provided as part of L<CCM::TextRender> since 15.10).
```

Comment:
define "old fucntionality". the old functionality relies on an external perl library `Config::General`, but the output is not reproducable (no sorting of keys and all that) and has some strange quirks too (https://github.com/quattor/CAF/issues/108 and refereneces therein). this leads to not being able to run unittests reliably, which is a problem.
so the most common part of the `Config::General` format is now produced using TT in (https://github.com/quattor/CCM/pull/73/files#diff-d8db3099576a834ae6c4de684c93ec08R1 in PR https://github.com/quattor/CCM/pull/73) and is called `general`. However, the TT code relies on some features that are made available via the internals of `CCM::TextRender` (not explaining this here, we can try to setup a confcall if needed. Strictly speaking, the parts required to make the TT work, could be ported back to CAF, but at the expense of people being confused and not knowing how to use it). My overall recommendation is not to use `CAF::TextRender` directly in any quattor code at all, use `CCM::TextRender` instead (https://github.com/quattor/quattor.github.com/blob/master/_posts/2015-09-20-textrender-basics.md#textrender-caf-and-ccm, esp the last remark of that paragraph). ncm-metaconfig and a whole bunch of other components already do this. 

So `CAF::TextRender` does not provide the `general` module anymore. If you want to use it, you can, but you have to use `CCM::TextRender`. So if you have a component that uses the general format, you have to do `s/CAF::TextRender/EDG::WP4::CCM::TextRender/g', for usage of general via metaconfig, nothing is required; this is assuming https://github.com/quattor/CCM/pull/73 also gets merged in the same release.

I'll adapt the formulation if you agree:
```perl
(Previously available module <general> was removed in 15.12. Component writers needing this functionality can use the L<CCM::TextRender> subclass instead).
```
(I'm afraid the conversation will get lost once i push a commit that modifies this line again.)